core/new-gui/src/app/workspace/service/joint-ui/joint-ui.service.ts

```diff
@@ -16,6 +16,7 @@ import {
   OperatorState,
   OperatorStatistics
 } from "../../types/execute-workflow.interface";
+import { CacheStatus } from "../../types/workflow-websocket.interface";
```

Comment:
I feel the name `CacheStatus` is too general. We should name it specific to `Result` or `OperatorResult`